src/main/service/ClientService.java

```diff
@@ -0,0 +1,32 @@
+package main.service;
+
+import main.entity.Ticket;
+import main.entity.User;
+import main.service.abstracts.AbstractUser;
+
+public class ClientService extends AbstractUser {
```

Comment:
why it is service?

Reply:
I just implemented it from Springboot
Spring Boot’s service layer pattern and the Separation of Concerns principle.

Reply:
We haven't touch yet SpringBoot :)

Service classes are used to write business logic and client is a part of model, with which objects service should use